bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/db/JdbcSqliteDAO.java

```diff
@@ -66,6 +68,15 @@ private void initDbProps() {
     /**************
      * ITEMS DAOs *
      **************/
+
+    @Override
+    public String doGetDB() {
+        // return Yank.queryScalar(SQL_GET_DB, String.class, null);
+        // List<String> g = Yank.queryColumn(SQL_GET_DB, "file", String.class, null).get(0);
+        // System.out.println("3 -- "+g.get(0));
```

Comment:
Thanks @lewie, could you remove the commented code if it's not providing a documentation function?